src/software/pipeline/main_cameraInit.cpp

```diff
@@ -438,6 +448,39 @@ int aliceVision_main(int argc, char **argv)
     const double diag24x36 = std::sqrt(36.0 * 36.0 + 24.0 * 24.0);
     camera::EIntrinsicInitMode intrinsicInitMode = camera::EIntrinsicInitMode::UNKNOWN;
 
+    std::unique_ptr<oiio::ImageInput> in(oiio::ImageInput::open(view.getImagePath()));
+
+    std::string imgFormat = in->format_name();
+
+    if (dcpDatabase.empty() && (imgFormat.compare("raw") == 0) && errorOnMissingColorProfile)
+    {
+        ALICEVISION_LOG_ERROR("The specified DCP database for color profiles does not exist or is empty.");
+        // No color profile available
+        boost::atomic_ref<char>{allColorProfilesFound} = 0;
+    }
```

Comment:
It only makes sense if we use DCP, isn't it?
If yes, it should be inside an if:
```
if((rawColorInterpretation == image::ERawColorInterpretation::DcpLinearProcessing ||
        rawColorInterpretation == image::ERawColorInterpretation::DcpMetadata)
{
    std::unique_ptr<oiio::ImageInput> in(oiio::ImageInput::open(view.getImagePath()));
    std::string imgFormat = in->format_name();
    if (dcpDatabase.empty() && (imgFormat.compare("raw") == 0) && errorOnMissingColorProfile)
    {
        ALICEVISION_LOG_ERROR("The specified DCP database for color profiles does not exist or is empty.");
        // No color profile available
        boost::atomic_ref<char>{allColorProfilesFound} = 0;
    }
}
```
And the message says that the file does not exist, but I do not see a test for that.

Reply:
The test for is dcpDatabase.empty(). dcpDatabase is loaded before entering in the parallel for. The database is empty if the directory given as parameter does not contain at least a dcp file.